FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c

```diff
@@ -1095,7 +1095,7 @@ void test_coverage_prvCreateIdleTasks_name_within_max_len( void )
 
     /* Test Verifications */
     xIdleTask = ( TCB_t * ) xIdleTaskHandles[ 0 ];
-    TEST_ASSERT_EQUAL_STRING( configIDLE_TASK_NAME, xIdleTask->pcTaskName );
+    TEST_ASSERT_EQUAL_STRING( "IDLE longX0", xIdleTask->pcTaskName );
```

Comment:
```suggestion
    TEST_ASSERT_EQUAL_STRING( "IDLE longXX", xIdleTask->pcTaskName );
```

Reply:
The tests will be fixed after this PR is merged - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203.